patterns/1-initial/improve-findability.md

```diff
@@ -56,9 +56,8 @@ None known as of yet---this is a pattern idea until it is proven.
 
 ## Status
 
-Unproven
-
-Brainstormed pattern idea reviewed 2017-03-11.
+* Initial  
+* Brainstormed pattern idea reviewed 2017-03-11.
```

Comment:
Idea: how about we add a _Revision History_ section, @spier , @lenucksi ? I feel it's kinda odd to have more than one entry in the _Status_ section. Just my 0.02 $.

Reply:
Captured it in https://github.com/InnerSourceCommons/InnerSourcePatterns/issues/269